cgmanifests/generated/cgmanifest.json

```diff
@@ -246,7 +246,7 @@
       "component": {
         "type": "git",
         "git": {
-          "commitHash": "ca678952a9a8eaa6de112d154e8e104b22f9ab3f",
+          "commitHash": "8a1772a0c5c447df2d18edf33ec4603a8c9c04a6",
```

Comment:
does https://github.com/microsoft/onnxruntime/blob/main/cmake/patches/cpuinfo/9bb12d342fd9479679d505d93a478a6f9cd50a47.patch need any update?

Reply:
It still works. We do not have to.

Reply:
The patch still can be applied cleanly.